File: client/src/androidTest/java/com/jraska/github/client/MakeTestsPassRule.java
package com.jraska.github.client;

import android.support.test.rule.ActivityTestRule;

import com.jraska.github.client.ui.UsersActivity;

/**
 * Currently we have to always start activity first
 */
public class MakeTestsPassRule extends ActivityTestRule<UsersActivity> {
  private MakeTestsPassRule() {
    super(UsersActivity.class);
  }

  public static MakeTestsPassRule create(){
    return new MakeTestsPassRule();
  }
}

File: client/src/androidTest/java/com/jraska/github/client/MakeTestsPassRule.kt
package com.jraska.github.client

import android.support.test.rule.ActivityTestRule

import com.jraska.github.client.ui.UsersActivity

/**
 * Currently we have to always start activity first
 */
class MakeTestsPassRule : ActivityTestRule<UsersActivity>(UsersActivity::class.java)
